contracts/SnappBase.sol

```diff
@@ -270,13 +287,4 @@ contract SnappBase is Ownable {
         // There is no situation where contract balance can't afford the upcomming transfer.
         ERC20(tokenIdToAddressMap[tokenId]).transfer(accountToPublicKeyMap[accountId], amount);
     }
-
-    function getDepositCreationBlock(uint slot) public view returns (uint) {
```

Comment:
Can we please leave them in for now? I could not find a way to read struct parameters with the web3 rust library. In js it's easy, but with rust, I was not finding any solution for 2 days...

Once, we have a solution, I am perfectly fine with removing these lines